src/main/include/Traj.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <math.h>
+
+#include "Constants.h"
+
+double getAngle(const double velocity, const double distance, const double height);
+
+bool isInRange(const double velocity, const double distance, const double angle);
+
+double getDistance(const double ll_angle, const double ll_height, const double target_height);
```

Comment:
May I also be a stickler about the naming convention in addition to spelling things out? 😬 You might also consider adding units to your measurement parameter names:

```suggestion
double GetAngle(const double velocity_mps, const double distance_meters, const double height_meters);

bool IsInRange(const double velocity_mps, const double distance_meters, const double angle_degrees);

double GetDistance(const double limelight_angle_degrees, const double limelight_height_meters, const double target_height_meters);
```

A potential improvement might be to use the units library, which would enforce the units for you:

```cpp
double GetDistance(const units::angle limelight_angle, const units::length limelight_height, const units::length target_height);
```

Just a thought.

Reply:
I am not personally familiar with the units library and what advantages and capabilities it has, so I'm going to stick with a primitive type for now.